pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -348,7 +348,9 @@ stdenv.mkDerivation ({
 
   passthru = passthru // {
 
-    inherit pname version;
+    inherit pname version ghc;
```

Comment:
I'd rather not export an attribute called `ghc` to denote the compiler. There are plenty of other Haskell compilers and even though we don't use them much now, we might in the future. Maybe this would better be called `compiler`?